docs/src/main/paradox/overview.md

```diff
@@ -1,21 +1,32 @@
 # Overview
 
-The Akka Persistence Cassandra plugin allows for using [Apache Cassandra](https://cassandra.apache.org) as a backend for @extref:[Akka Persistence](akka:persistence.html) and @extref:[Akka Persistence Query](akka:persistence-query.html).
+The Akka Persistence Cassandra plugin allows for using [Apache Cassandra](https://cassandra.apache.org) as a backend for @extref:[Akka Persistence](akka:persistence.html) and @extref:[Akka Persistence Query](akka:persistence-query.html). It uses @extref:[Alpakka Cassandra](alpakka:cassandra.html) for Cassandra access which is based on the @extref:[Datastax Java Driver](java-driver:).
 
 ## Project Info
 
 @@project-info{ projectId="core" }
 
 ## Dependencies
 
+This plugin requires **Akka 2.6** and is built against Akka $akka.version$.
```

Comment:
or
```
This plugin requires **Akka $akka.version$** or later.
```

since users don't understand that they can use later version than the build version.